app/services/bulk_email_sender_service.rb

```diff
@@ -1,6 +1,6 @@
 class BulkEmailSenderService
-  def self.call(subject:, body:, subscriber_lists:, queue: :delivery_immediate)
-    email_ids = BulkEmailBuilder.call(subject: subject, body: body, subscriber_lists: subscriber_lists).ids
+  def self.call(bulk_email_builder:, queue: :delivery_immediate)
```

Comment:
We also need to change [this task](https://github.com/alphagov/email-alert-api/blob/master/lib/tasks/bulk.rake). I was tempted to remove it, but [it's an official thing](https://docs.publishing.service.gov.uk/manual/bulk-email.html). Maybe we should add a test for it, so it won't break in the future.

Reply:
Ah good spot - thanks! Yes will add a test.

Reply:
@benthorner [added a test for this rake task](https://github.com/alphagov/email-alert-api/pull/1261/commits/4c9951f5050a65629cb53b5138733389fe7fb19f)